CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.17.0 Release
+- Release Date: March 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
```

Comment:
It seems we're not doing in the past but should we do the same compatibility notice for iOS device and min. OS version?